citesphere/src/main/java/edu/asu/diging/citesphere/api/v1/user/CitationsApiController.java

```diff
@@ -0,0 +1,73 @@
+package edu.asu.diging.citesphere.api.v1.user;
+
+import java.io.IOException;
+import java.security.Principal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.asu.diging.citesphere.api.v1.V1Controller;
+import edu.asu.diging.citesphere.core.service.ICitationManager;
+import edu.asu.diging.citesphere.core.user.IUserManager;
+import edu.asu.diging.citesphere.model.bib.ICitationGroup;
+import edu.asu.diging.citesphere.model.bib.impl.CitationResults;
+import edu.asu.diging.citesphere.user.IUser;
+
+@Controller
+public class CitationsApiController extends V1Controller {
+	
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Value("${_api_page_size}")
+    private Integer apiPageSize;
+
+    @Value("${_available_item_columns}")
+    private String availableColumns;
+
+    @Autowired
+    private ICitationManager citationManager;
+
+    @Autowired
+    private IUserManager userManager;
+    
+    @Autowired
+    private ObjectMapper objectMapper;
+	
+    @RequestMapping(value = {"/citations/search"}, produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
since this is not a general search but searches for citations with a specific contributor, the url needs to reflect that.